test/switch.civet

```diff
@@ -629,3 +629,68 @@ describe "switch", ->
       else if(x instanceof Bar) {
           console.log("it's a Bar")}
     """
+
+    testCase """
+      implicit return
+      ---
+      ->
+        switch x
+          1
+            2
+          else
+            3
+      ---
+      (function() {
+        if(x === 1) {
+            return 2}
+      else  {
```

Comment:
Is the indentation off here? Maybe it's getting inserted after the `else` instead of before?

Reply:
Yeah, the indentation is way off on the else branches currently.